### PR TITLE
Add a summary of our third-party dependencies

### PR DESCRIPTION
**Pull request by badboy**
I will document the exact process in a followup

### CODE REVIEW

**Comment by mdboom** [comment]
IIUC, this extracts the Rust dependencies, but not Kotlin, Swift or Python dependencies.  Should we file bugs to tackle those?  (It's probably less critical, since there's no GPL-triggering static linking possible in Kotlin or Python, but it would still be nice to be complete...)

File: publish.gradle [hunk 1]
```diff
@@ -103,6 +103,15 @@ ext.configurePublish = { jnaForTestConfiguration = null ->
                         }
                     }
 
+                    def depLicenses = new XmlSlurper().parse(new File("${projectDir}/dependency-licenses.xml"))
```
Comment (Dexterp37) [inline publish.gradle]: NEAT! So this gets added to our artefacts. Nice.

**Comment by badboy** [comment]
Will file one.
The nice thing: cargo-about is extendeable and you can add "custom" licenses for other code!

File: publish.gradle (hunk 1 above)
Comment (badboy) [reply publish.gradle]: Yes, copied over from a-s :D

**Comment by badboy** [comment]
Filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1602447

**Pull request merged**